gateway/src/connectors/defira/defira.ts

```diff
@@ -14,36 +14,20 @@ import {
 } from '@ethersproject/contracts';
 import {
   // TODO: use this instead of UniswapRouter
-  Router,
+  Router as UniswapRouter,
```

Comment:
Don't call these `UniswapXXX`. I named them that way to denote they're not applicable.

Reply:
It doesn't make sense to call these Uniswap when they're coming from Defira SDK. TODO was literally saying to replace instances of `UniswapXXX` with `XXX`